src/lib/selection/media/html-selector.js

```diff
@@ -63,7 +63,7 @@ export default class HTMLSelector extends MediaSelector {
     textSelector.model = LanguageModelFactory.getLanguageModel(this.languageID)
     textSelector.location = this.location
     textSelector.data = this.data
-    if (this.browserSelector) {
+    if (this.browserSelector && this.languageID !== Constants.LANG_CHINESE) {
```

Comment:
Can we add a TODO here? We will want the data-alpheios-word-node functionality to work eventually with chinese but I agree it isn't necessary for the prototype

Reply:
Second to that

Reply:
I have added TODO
I believe that such attribute could be used only with embed-lib, in webextension - we would need to use language anyway.

Reply:
Oh you have some other ideas?

Reply:
> I have added TODO
> I believe that such attribute could be used only with embed-lib, in webextension - we would need to use language anyway.

Not sure what you mean here.   The way this attribute is intended to be used is as follows (from the faq):

The data-alpheios-word-node attribute can be used to indicate that an element contains a complete word, and the text should be that of the entire node, including the child node text. The default value of default means that the word will be treated to default Alpheios word cleanup behavior, which includes stripping language-specific punctuation (of which the standard Leiden Conventions punctuation marks are included for Latin and Greek). A value of exact can be used to specify that the text contents should be taken exactly as supplied (excluding node markup).

For example:

```
<span data-alpheios-word-node="default"><b>f</b>ero</span> indicates the word should be analyzed as 'fero'
<span data-alpheios-word-node="default">f[ero</span> will also be analyzed as 'fero'
<span data-alpheios-word-node="exact">f[ero</span> will be analyzed as 'f[ero'
```

I agree it's unlikely that people will markup their pages like this unless they are using the embedded library, but it would work with the webextension if they did.